src/markdown/tables.scss

```diff
@@ -13,13 +11,12 @@
 
     th,
     td {
-      padding: 6px 13px;
-      border: 1px solid lighten($gray-300, 5%);
+      padding: ($spacer-2 - 2) ($spacer-3 - 2);
+      border: $border-width $border-style $border-gray-dark;
     }
 
     tr {
       background-color: $bg-white;
-      border-top: 1px solid darken($gray-300, 4%);
```

Comment:
What's the reasoning for removing the top border here?

Reply:
Was wondering the same, but removing it in DevTools doesn't seem to have an effect? Maybe because `td`s already have a border? 🤔

Reply:
Ohh.. it says it in the description:

> * `tr` elements don't render CSS borders, so I've removed the `border` declaration